easybuild/easyconfigs/h/HOME/HOME-0.9_setup.patch

```diff
@@ -0,0 +1,14 @@
+diff -ru HOME-0.9/setup.py HOME-0.9.modified/setup.py
```

Comment:
@timeu Please include a short description of the patch + mention author or upstream reference (and then fix the SHA256 checksum accordingly)